app/window.py

```diff
@@ -896,6 +896,57 @@ def setTextBuffer(self, textBuffer):
     Window.setTextBuffer(self, textBuffer)
     self.controller.setTextBuffer(textBuffer)
 
+class PopupWindow(Window):
+  def __init__(self, host):
+    assert(host)
+    Window.__init__(self, host)
+    self.host = host
+    self.controller = app.cu_editor.PopupController(self)
+    self.setTextBuffer(app.text_buffer.TextBuffer())
+    self.controller.setTextBuffer(self.textBuffer)
```

Comment:
consider overriding setTextBuffer and set the controller.setTextBuffer there. It doesn't make a big difference right now, but maybe in the future: if if setTextBuffer is called it should tell the controller.

Reply:
Sure, sounds good.